### DISCUSSION TITLE
20221028/redis-20221028-NoSQL%E6%95%B0%E6%8D%AE%E5%BA%93%E7%AE%80%E4%BB%8B/

### DISCUSSION THREAD

**Discussion by giscus** [renmcc/renmcc#1] [General]
# 20221028/redis-20221028-NoSQL%E6%95%B0%E6%8D%AE%E5%BA%93%E7%AE%80%E4%BB%8B/

1. 技术发展  解决功能性问题：Java、Jsp、RDBMS、Tomcat、HTML、Linux、JDBC、SVN 解决扩展性问题：Struts、Spring、SpringMVC、Hibernate、Mybatis 解决性能的问题：NoSQL、Java线程、Hadoop、Nginx、MQ、ElasticSearch  1.1 Web1.0时代 Web1.0的时代，数据访问量很有限、用一夫当关的高

http://localhost:4000/20221028/redis-20221028-NoSQL%E6%95%B0%E6%8D%AE%E5%BA%93%E7%AE%80%E4%BB%8B/

**Comment by renmcc** [comment]
测试
